airbyte_cdk/sources/declarative/extractors/record_filter.py

```diff
@@ -11,7 +10,7 @@
     PerPartitionWithGlobalCursor,
 )
 from airbyte_cdk.sources.declarative.interpolation.interpolated_boolean import InterpolatedBoolean
-from airbyte_cdk.sources.types import Config, StreamSlice, StreamState
+from airbyte_cdk.sources.types import Config, StreamSlice, StreamState, Record
```

Comment:
nit: is ruff validating order of import? It seems like it should be alphabetical here

Reply:
added sorting option in https://github.com/airbytehq/airbyte-python-cdk/pull/94